package.json

```diff
@@ -1,33 +1,40 @@
 {
+  "packageManager": "yarn@3.2.0",
+  "workspaces": [
+    "packages/*"
+  ],
   "name": "root",
   "private": true,
   "engines": {
     "node": ">=14.0.0 <17.0.0"
   },
   "scripts": {
+    "build": "yarn workspaces foreach -ptv run build",
+    "build:public": "yarn workspaces foreach -ptv --no-private run build",
     "lint": "eslint . --ext .js,.ts,.vue --max-warnings=0",
     "prettier:check": "prettier --check .",
     "prettier:fix": "prettier --write .",
     "docker:deps:up": "docker-compose -f ./docker-compose-deps.yml up -d",
     "docker:deps:down": "docker-compose -f ./docker-compose-deps.yml down",
-    "dev": "lerna run dev --parallel",
-    "dev:no-server": "npm run dev -- --ignore @speckle/server",
+    "dev": "yarn workspaces foreach -ptv --exclude @speckle/frontend run build:dev && yarn workspaces foreach -pv -j unlimited run dev",
```

Comment:
Can you elaborate on this change? Previously we ran 'dev' on all packages

Reply:
Does running dev on the objectloader build it into the dist folder? That was my intention here, to make sure viewer and objectloader is properly built, before running dev on everything.

Reply:
It should yea. I mean on the current main where we still have npm+lerna we're running `dev` in all packages IIRC, and it seems to work ok, so unless if something in the build process changed in this PR then I'd think it would still work.

Reply:
I'll test this before merging